cpp/src/arrow/compute/kernels/filter_test.cc

```diff
@@ -468,5 +468,157 @@ TEST_F(TestFilterKernelWithUnion, FilterUnion) {
   }
 }
 
+class TestFilterKernelWithRecordBatch : public TestFilterKernel<RecordBatch> {
+ public:
+  void AssertFilter(const std::shared_ptr<Schema>& schm, const std::string& batch_json,
```

Comment:
nit: not fond of condensed variable names like `schm`

Reply:
If I don't do something like this, I collide with the function named `schema`:

```
/Users/enpiar/Documents/ursa/arrow/cpp/src/arrow/compute/kernels/filter_test.cc:492:17: error: variable 'schema' declared with deduced type 'auto' cannot appear in its own initializer
  auto schema = schema(fields);
                ^
```